.travis.yml

```diff
@@ -24,13 +24,15 @@ before_install:
   - pip install -U pip wheel
   - pip install $DJANGO
   - pip install -U -r requirements-test.txt
+  # temporary: remove when openwisp-utils 0.5.0 is released
+  - pip install -U https://github.com/openwisp/openwisp-utils/tarball/master#egg=openwisp_utils[qa]
 
 install:
   - python setup.py -q develop
-
+  
 before-script:
   - >
-    openwisp-utils-qa-checks \
+    openwisp-qa-check \
```

Comment:
can you please create a `run-qa-checks` script as in the other repos? Look at openwisp-monitoring.

Reply:
remember this :point_up: